_docs/03.8-server_side_data_hydration.markdown

```diff
@@ -0,0 +1,66 @@
+---
+title:  "Server Side Data Hydration"
+permalink: docs/ss_data_hydration.html
+toplevel: "Getting Started: Intermediate"
+---
+The server side rendering in Electrode consists of two tasks, the actual rendering of content to html and the initial state loading into the redux store(server side data hydration).
+Server side data hydration refers to instantiating the redux store on the server side. This data is then passed down to the client as a part of the initial call. Thus, all data pertaining to the initial state of the app is available to the without making any additional API calls to the server.
```

Comment:
Server side data hydration refers to passing the redux initial store data that was used for `ReactDom.renderToString` being passed to the browser so react can use the same data to bootstrap rendering on the client side.  This helps the client side avoid making additional calls to the server to retrieve data for rendering.